onnxscript/ir/_core.py

```diff
@@ -1924,6 +1924,10 @@ def __init__(
         # Be sure the initialize the name authority before extending the nodes
         # because it is used to name the nodes and their outputs
         self._name_authority = _name_authority.NameAuthority()
+        # NOTE: input and initializer value names could be duplicated
+        # to auto-generated value names from name authority and crash ort
+        # https://github.com/microsoft/onnxruntime/blob/bc7b07dbb41a2f441dbed1a91855563ba0dd8a31/onnxruntime/core/graph/graph.cc#L1536
```

Comment:
```suggestion
```
I assume the following line fixes this problem? We could update the comment to explain what is accomplished by the following line instead.